packages/ui-map-components/src/components/Table/getMapTableData.tsx

```diff
@@ -13,22 +13,26 @@ const FirstColumnCell = styled.span`
 `;
 
 const processColumns = (serieses: Series[]) => {
+  console.log(serieses);
   if (!serieses) {
     return [];
   }
 
   const configColumns = serieses.map(column => {
-    return { accessor: column.key, Header: column.name };
+    return {
+      accessor: (row: any) => row[column.key],
```

Comment:
Would you mind add the comment in here - just in case anyone puts it back to what it was before accidentally :)